packages/@uppy/tus/src/index.js

```diff
@@ -215,6 +215,22 @@ module.exports = class Tus extends BasePlugin {
         if (typeof opts.onBeforeRequest === 'function') {
           opts.onBeforeRequest(req)
         }
+
+        if (queuedRequest == null) {
```

Comment:
This good definitely use an extensive comment on what is going on here and why it works. It seems a bit random if you don't know the context